Document nested group support for AsyncAPI (#5682)
mintlify/docs: api-playground/asyncapi-setup.mdx

  The `directory` field is optional. If not specified, Mintlify adds the files to the **api-reference** folder of the docs repository.
</Note>

### Examples with nested groups

> You can also add the `asyncapi` property to groups nested inside other groups. Mintlify generates the channel pages and adds them to the nested group, alongside any existing pages.

Reviewer: ```suggestion
The `asyncapi` property supports nested groups. Mintlify generates the channel pages and adds them to the nested group, alongside any existing pages.
```
